.github/workflows/sub_package_update.yml

```diff
@@ -75,5 +75,10 @@ jobs:
           branch-suffix: timestamp
           title: ${{ matrix.title }}
           body: |
+            This is an autogenerated PR, which will ${{ matrix.body }}.
+            [Cruft](https://cruft.github.io/cruft/) has detected updates from the Package Template.
```

Comment:
you don't need to duplicate these lines. you should add the labels line at the same level of indentation as `body` at the end of the file.

Reply:
I have removed duplicated lines.